data/json/furniture.json

```diff
@@ -347,6 +363,7 @@
         "name": "wood stove",
         "symbol": "#",
         "bgcolor": "red",
+        "description": "Wood stove for heating and cooking. Good thing the inside isn't made of wood.",
```

Comment:
Could use a line mentioning it's more efficient than burning things in the open.